excel/beginner/basics-of-excel/what-is-a-workbook.md

```diff
@@ -0,0 +1,41 @@
+---
+author: Stefan-Stojanovic
+
+levels:
+  - beginner
+
+aspects:
+  - new
+  - workout
+
+type: normal
+
+category: how to
+
+---
+
+# What is a workbook?
+
+---
+## Content
+
+A workbook is a single file containing one or more worksheets(sheets).
+
+When you create a new workboot, by default, you get 1 worksheet. More sheets can be added at any time.
+
+The main difference between a workbook and a worksheet(sheet) is that you can have many worksheets inside a single workbook, but not vice versa. The workbook is the main file and it can contain as many worksheets as needed.

```

Comment:
```suggestion
The main difference between a workbook and a worksheet (sheet) is that you can have many worksheets inside a single workbook, but not vice versa. The workbook is the main file and it can contain as many worksheets as needed.

```